cms/models/pagemodel.py

```diff
@@ -354,7 +354,7 @@ def _copy_contents(self, target, language):
                 new_phs.append(ph)
                 # update the page copy
             if plugins:
-                copy_plugins_to(plugins, ph, no_signals=True)
```

Comment:
@mkoistinen Good point. I will change ``CMSPlugin.copy_plugin`` to remove the no_signal and instead always explicitly mark the placeholder cache as dirty. Will do so in a separate pr. Ok with you?